nikolay.braslavets/hw02/tamagochi.rb

```diff
@@ -0,0 +1,169 @@
+class Tamagochi
+  COMMANDS = %w(help feed play sleeping heal speak praise scold poop scratch claws kicked stat)
```

Comment:
https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/MutableConstant
Please freeze values assigned to constants
Prefer [] for %w definitions